OlinkAnalyze/R/olink_qc_plot.R

```diff
@@ -1,27 +1,53 @@
 #' Function to plot an overview of a sample cohort per Panel
 #'
 #' Generates a facet plot per Panel using ggplot2::ggplot and ggplot2::geom_point and stats::IQR plotting IQR vs. median for all samples.
-#' Horizontal dashed lines indicate +/-3 standard deviations from the mean IQR.
-#' Vertical dashed lines indicate +/-3 standard deviations from the mean sample median.
+#' Horizontal dashed lines indicate +/-IQR_outlierDef standard deviations from the mean IQR (default 3).
```

Comment:
May want to add some error handling here as well to check for T/F, since that will cause them to be set to 1/0 in both IQR_outlierDef and median_outlierDef

Reply:
I agree